app/pages/statistics.tsx

```diff
@@ -116,6 +111,31 @@ const getCountByDate = (
   return countByDate;
 };
 
+const getTrend = (
+  data: PageProps["countByDay"]
+): {
+  direction: "up" | "down";
+  lastMonthDailyAverage: number;
+  previousThreeMonthsDailyAverage: number;
+} => {
+  const now = Date.now();
+  const nowDate = new Date(now);
+  const lastMonthDailyAverage =
+    data.filter(({ day }) => {
+      return day > new Date(now - 30 * DAYS_IN_MS) && day <= nowDate;
+    }).length / 30;
+  const previousThreeMonthsDailyAverage =
+    data.filter(({ day }) => {
+      return day > new Date(now - 90 * DAYS_IN_MS) && day <= nowDate;
+    }).length / 90;
+  return {
+    direction:
+      lastMonthDailyAverage > previousThreeMonthsDailyAverage ? "up" : "down",
+    lastMonthDailyAverage,
+    previousThreeMonthsDailyAverage,
+  };
+};
```

Comment:
For what's it's worth, the averages could also be directly be computed from the db

```
WITH
    last_month_daily_average AS (
        SELECT COUNT(*) / 30.0 AS daily_average
        FROM config
        WHERE
            created_at > CURRENT_DATE - INTERVAL '30 days'
            AND created_at <= CURRENT_DATE
    ),
    last_three_months_daily_average AS (
        SELECT COUNT(*) / 90.0 AS daily_average
        FROM config
        WHERE
            created_at > CURRENT_DATE - INTERVAL '90 days'
            AND created_at <= CURRENT_DATE
    )
SELECT
    (SELECT daily_average FROM last_month_daily_average) AS last_month_daily_average,
    (SELECT daily_average FROM last_three_months_daily_average) AS previous_three_months_daily_average;
```

Reply:
Thanks 💯 Done in https://github.com/visualize-admin/visualization-tool/pull/1610/commits/f234cb409c377857cf351fcb877de89040f7b29f 👍